nekoyume/Assets/_Scripts/BlockChain/ActionRenderHandler.cs

```diff
@@ -2149,5 +2161,131 @@ private void ResponseClaimRaidReward(ActionBase.ActionEvaluation<ClaimRaidReward
             WorldBossStates.SetReceivingGradeRewards(avatarAddress, false);
             Widget.Find<WorldBossRewardScreen>().Show(new LocalRandom(eval.RandomSeed));
         }
+
+        private void ResponseBattleGrandFinale(ActionBase.ActionEvaluation<BattleGrandFinale> eval)
```

Comment:
How about you reusing same logic with `ResponseBattleArena()` method?

Reply:
I think that's a good suggestion. However, I think it is good to manage it even if it overlaps right now.